pkg/sdn/api/validation/validation.go

```diff
@@ -84,14 +84,3 @@ func ValidateNetNamespace(netnamespace *sdnapi.NetNamespace) fielderrors.Validat
 	}
 	return allErrs
 }
-
-func ValidateNetNamespaceUpdate(obj *sdnapi.NetNamespace, old *sdnapi.NetNamespace) fielderrors.ValidationErrorList {
```

Comment:
add back, remove the `obj.NetID != old.NetID` if that is now allowed to be updated, but delegate to `ValidateNetNamespace` to check valid NetID (non-negative)